src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -690,18 +690,22 @@ def add_fluorescence_traces(
 
     if "dff" in traces_to_add:
         df_over_f_data_interface = _get_segmentation_data_interface(nwbfile=nwbfile, data_interface_name=df_over_f_name)
-        trace_to_data_interface.update(Dff=df_over_f_data_interface)
+        trace_to_data_interface.update(dff=df_over_f_data_interface)
 
     for trace_name, trace in traces_to_add.items():
+        # Decide which data interface to use based on the trace name
+        data_interface = trace_to_data_interface[trace_name]
         # Extract the response series metadata
-        trace_name = "RoiResponseSeries" if trace_name == "raw" else trace_name.capitalize()
+        # The name of the roi_response_series is "RoiResponseSeries" for raw and df/F traces,
+        # otherwise it is capitalized trace_name.
+        trace_name = "RoiResponseSeries" if trace_name in ["raw", "dff"] else trace_name.capitalize()
```

Comment:
@bendichter 
This is an important change after the last [commit](https://github.com/catalystneuro/neuroconv/pull/41/commits/72104e7810b19fd85826713a9506722ee95b1a1e), that we will write the df/F trace under the name of "RoiResponseSeries". I know this is far from ideal, before it was expected that it is "Dff" and hence the tests started failing. The complication is that we are renaming the trace names here raw->RoiResponseSeries and then we are matching the metadata with this name. But i'm not sure how to rely less on the name of the trace when assigning the metadata to it.

Reply:
Seems highly analogous to the role of `es_key` in the `ecephys` `add_electrical_series` function: https://github.com/catalystneuro/neuroconv/blob/main/src/neuroconv/tools/spikeinterface/spikeinterface.py#L561

For any given `RoiResponseSeries`, I'd think we should therefore follow a similar practice to allow identification of which part of the metadata to use when writing a given response series using a user-definable key.

How does that sound? Would you be able to do that in a separate PR so we can at least get this one in sooner?

Reply:
Similarly, the assigned default name could follow the same practice of appending `RoiResponseSeries` with the type, such as `RoiResponseSeriesDfOverF` or `RoiResponseSeriesRaw`, etc. like in https://github.com/catalystneuro/neuroconv/blob/main/src/neuroconv/tools/spikeinterface/spikeinterface.py#L630

Reply:
Let's try to follow our [naming conventions](https://nwbinspector.readthedocs.io/en/dev/best_practices/general.html#naming-conventions)

If there's just one RoiResponseSeries, it should be called "RoiResponseSeries"

Reply:
Well sure if there's only one then the extra naming isn't necessary. The additional values would be context in the event that there are more than one

Reply:
Yes, I see your point, so instead of saving them as "RoiResponseSeries" and "Deconvolved" if we have raw and deconvolved traces, we'll eventually want to write as RoiResponseRaw and RoiResponseSeriesDeconvolved? but only  "RoiResponseSeries"  if only one trace is there. Yeah, it should be done in a separate PR